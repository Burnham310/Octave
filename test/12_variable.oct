main = { c1=[1 3 5], c2=[2 4 6], c5=[5 7 ^1]; bpm=60;
    c2 c5 c1 []
    c2 c5 c1 c1
}
