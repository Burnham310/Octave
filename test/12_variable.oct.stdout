[
{
  "freq": 146.82925415039062,
  "duration": 1,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 174.6103973388672,
  "duration": 1,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 219.9953155517578,
  "duration": 1,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 0.000000000000000000000011373491242306228,
  "duration": 1,
  "gap": 1,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 195.99354553222656,
  "duration": 1,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 246.93638610839844,
  "duration": 1,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 261.6199951171875,
  "duration": 1,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 0.000000000000000000000011373491242306228,
  "duration": 1,
  "gap": 1,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 130.80999755859375,
  "duration": 1,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 164.81027221679688,
  "duration": 1,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 195.99354553222656,
  "duration": 1,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 0.000000000000000000000011373491242306228,
  "duration": 1,
  "gap": 1,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 0.000000000000000000000011373491242306228,
  "duration": 1,
  "gap": 1,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 146.82925415039062,
  "duration": 1,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 174.6103973388672,
  "duration": 1,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 219.9953155517578,
  "duration": 1,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 0.000000000000000000000011373491242306228,
  "duration": 1,
  "gap": 1,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 195.99354553222656,
  "duration": 1,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 246.93638610839844,
  "duration": 1,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 261.6199951171875,
  "duration": 1,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 0.000000000000000000000011373491242306228,
  "duration": 1,
  "gap": 1,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 130.80999755859375,
  "duration": 1,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 164.81027221679688,
  "duration": 1,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 195.99354553222656,
  "duration": 1,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 0.000000000000000000000011373491242306228,
  "duration": 1,
  "gap": 1,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 130.80999755859375,
  "duration": 1,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 164.81027221679688,
  "duration": 1,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 195.99354553222656,
  "duration": 1,
  "gap": 0,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 0.000000000000000000000011373491242306228,
  "duration": 1,
  "gap": 1,
  "amp": 0,
  "instrument": "acoustic-guitar"
}
,
{
  "freq": 0,
  "duration": 0,
  "gap": 0,
  "amp": 0,
  "instrument": "dummy"
}

]